docs/admin/serverless/serverless.md

```diff
@@ -2,15 +2,15 @@
 KServe Serverless installation enables autoscaling based on request volume and supports scale down to and from zero. It also supports revision management
 and canary rollout based on revisions.
 
-Kubernetes 1.22 is the minimally required version and please check the following recommended Knative, Istio versions for the corresponding
+Kubernetes 1.28 is the minimally required version and please check the following recommended Knative, Istio versions for the corresponding
 Kubernetes version.
 
 ## Recommended Version Matrix
 | Kubernetes Version | Recommended Istio Version | Recommended Knative Version |
 |:-------------------|:--------------------------|:----------------------------|
-| 1.27               | 1.18,1.19                 | 1.10,1.11                   |
-| 1.28               | 1.19,1.20                 | 1.11,1.12.4                 |
-| 1.29               | 1.20,1.21                 | 1.12.4,1.13.1               |
+| 1.28               | 1.22                      | 1.15                        |
+| 1.29               | 1.22,1.23                 | 1.15,1.16                   |
+| 1.30               | 1.22,1.23                 | 1.15,1.16                   |
```

Comment:
Knative has not released 1.16 yet

Reply:
knative is releasing 1.16 in a week, it is probably fine

Reply:
correct. today is the expected date :)